introduction_to_amazon_algorithms/object2vec_sentence_similarity/object2vec_sentence_similarity.ipynb

```diff
@@ -663,6 +663,65 @@
     "regressor.fit(input_channels)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Plot evaluation metrics for training job\n",
+    "\n",
+    "Evaluation metrics for the completed training job are available in CloudWatch. We can pull the cross entropy metric of the validation data set and plot it to see the performance of the model over time."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "import datetime\n",
+    "import matplotlib.pyplot as plt\n",
+    "import pandas as pd\n",
+    "\n",
+    "client = boto3.client('cloudwatch')\n",
+    "\n",
+    "latest_job_name = regressor.latest_training_job.job_name\n",
```

Comment:
Can you put the code in a function so that this can be reused for the other training tasks below? See the `How to enable the optimal training result` below. There is a regressor `new_regressor`